File: week2-Arrays/exit.c
#include <cs50.h>
#include <stdio.h>

int main(int argc, string argv [])
{
    if (argc != 2) // 2 bcos the one that is untyped is the first such that what i type is the second ie argv[1]
    {
        printf("missing command-line argument\n");
        return 1;
    }
    printf("hello, %s\n", argv[1]);
    return 0;
}

// make exit
// ./exit David

File: week6-Python/exit.py
from sys import argv, exit
if len(argv) != 2: #2 bcos the one that is untyped is the first such that what i type is the second ie argv[1]
    print ("missing command-line argument")
    exit(1)
print (f"hello, {argv[1]}")
exit(0)


#  python3 exit.py  -->> missing command-line argument
#  python3 exit.py emm maa me -->> missing command-line argument
#  python3 exit.py emma  -->> hello, emma